testdata/serviceconfig.yml

```diff
@@ -30,7 +30,17 @@ rules:
   - kind: access-logs
     params:
       logName: "access_log"
-      logFormat: 0 # Common Log Format
+      log:
+        log_format: 1 # apache common log format
+        template_expressions:
+           originIp: origin.ip
+           source_user: origin.user
+           timestamp: request.time
+           method: request.method | ""
```

Comment:
The others should already be reported by the proxy, but request.method (with its new meaning of HTTP method) it not yet implemented.